scripts/core/editor3/components/toolbar/index.tsx

```diff
@@ -126,6 +125,7 @@ class ToolbarComponent extends React.Component<any, IState> {
     }
 
     componentDidMount() {
+        this.scrollContainer = $(this.props.scrollContainer || window);
```

Comment:
It's better not to set a default. Otherwise it would make it impossible to have it non-sticky, and may break the UI for existing components that do not have it set are exist in another scroll container.